monailabel/endpoints/proxy.py

```diff
@@ -80,6 +82,11 @@ async def proxy_dicom(op: str, path: str, response: Response):
     return response
 
 
+@router.get("/dicom/{path:path}", include_in_schema=False)
```

Comment:
this is breaking the basic existing functionality (connecting to orthanc etc..)
We can't have multiple prefix for the same kind..
/dicom
/dicom/wado etc..